static-site/src/util/blogPosts.js

```diff
@@ -36,6 +42,76 @@ export function getBlogPosts() {
   return blogPosts;
 }
 
+/**
+ * Writes RSS2, Atom, and JSON feeds into the `public/blog` directory.
+ */
+export async function generateBlogFeeds() {
+  const feedDir = `${__dirname}/../../public/blog`;
+  const siteURL = "https://nextstrain.org/blog";
+
+  const feed = new Feed({
+    title: "Nextstrain.org Blog",
+    description: "FIXME",
```

Comment:
```suggestion
    description: "Updates from the Nextstrain core team",
```

Doesn't have to be this, just something other than `FIXME` 😄

Reply:
I've been waiting for somebody to suggest something! :grin: